apps/wappush/js/messagedb.js

```diff
@@ -243,7 +278,11 @@ var MessageDB = (function() {
           state.result = message;
```

Comment:
or is it necessary that we can retrieve a message even if it's in the viewed store so that we can hide it ?